src/components/calcite-action-pad/calcite-action-pad.scss

```diff
@@ -1,5 +1,6 @@
 :host {
   @extend %component-host;
+  --calcite-animation-timing: 300ms;
```

Comment:
Aside from focus timing that will go away, I thought all components would rely on the same global timing var. Am I missing something? cc @caripizza @macandcheese

Reply:
@jcfranco no I think you have it right. It's just that a few components which reference these global animation utility classes (ex `animate-in-up`) seem like they need the longer 300ms speed for their animations (not the new 150ms speed from updated the global var value). That's why I recommended that Anvesh just reset their values directly above the animation class as he's doing here.

Reply:
>  It's just that a few components which reference these global animation utility classes (ex animate-in-up) seem like they need the longer 300ms speed for their animations

I see. What are the downsides to having all of them use the global var? Would some animations look off? 

We need a way to allow users to override the global variable to tweak or even disable animations. Having component-specific overrides does defeats the purpose. I think @anveshmekala brought this up before. Looping in @asangma + @macandcheese for suggestions.

Reply:
These utility classes are using the global var already - is that your concern?
https://github.com/Esri/calcite-components/blob/master/tailwind.config.js#L130

Reply:
My concern having a mixed usage of the global var and some components overriding the duration var. I'd like to have users override a single global var for all animation/transition duration. Let me know if this makes sense. We can have a quick chat later today if I'm misunderstanding the approach.